index.js

```diff
@@ -53,6 +53,8 @@ module.exports = app => {
       name: core.getInput('name') || undefined
     })
 
+    const shouldPublish = core.getInput('publish') === 'true'

```

Comment:
what if I spelled it `True` or `TRUE`

Reply:
Good call. It looks like going case-insensitive is [consistent with Github’s own actions](https://github.com/actions/checkout/blob/f858c22e963bc60bc9d01c3d105c52a45a7deb6e/src/input-helper.ts#L85)